src/main/java/io/th0rgal/oraxen/items/OraxenMeta.java

```diff
@@ -51,6 +53,7 @@ public void setPackInfos(ConfigurationSection configurationSection) {
         // If not specified, check if a model or texture is set
         this.generate_model = configurationSection.getBoolean("generate_model", getModelName().isEmpty());
         this.parentModel = configurationSection.getString("parent_model", "item/generated");
+        this.modelDirectory = configurationSection.getString("model_directory", "minecraft/models");

```

Comment:
`minecraft/models` here is wrong, this should be just empty.
Modelpath = namespace:path and that path does not include the models/textures folder

Reply:
So how should i get correct path to generate model? By creating a method that will convert this `namespace:path` to normal file path?

Reply:
this should just be following the same pattern as model and textures
aka if u want ur model to go into assets/namespace/models/something/mymodel.json you put `namespace:something/mymodel`

Not entirely sure how I feel about this property anyway but most wont care to use it since there isnt much reason to but

Reply:
What do you think about it now?